File: Typescript/Practice/26072022/easy/easy_milkAndCookies.py
#!/usr/bin/env python

import datetime

def time_for_milk_and_cookies(the_date):
  if the_date.month == 12 and the_date.day == 24:
    return True
  else:
    return False

print(time_for_milk_and_cookies(datetime.date(2013, 12, 24)))
print(time_for_milk_and_cookies(datetime.date(2013, 1, 23)))
print(time_for_milk_and_cookies(datetime.date(3000, 12, 24)))

File: Typescript/Practice/easy/easy_milkAndCookies.ts
#!/usr/bin/env npx ts-node

function timeForMilkAndCookies(date: Date) {
  if (date.getMonth() === 11 && date.getDate() === 24) {
    return true;
  } else {
    return false;
  };
};

console.log(timeForMilkAndCookies(new Date(2013, 11, 24)));
console.log(timeForMilkAndCookies(new Date(2013, 0, 23)));
console.log(timeForMilkAndCookies(new Date(3000, 11, 24)));

// const date = new Date(2013, 11, 24)
// console.log(date.getDate());
